docs/source/classroom/class_guide.md

```diff
@@ -0,0 +1,172 @@
+
+## Classroom setup
+---
+
+Once access is provided, each course will have a course-specific directory under `/courses/` following this sample file tree. As shown for the course BINF6430.202410 below:
+
+:::{code-block} bash
+
+/courses/
+└── BINF6430.202410/
+    ├── data/
+    ├── staff/
+    └── students/
+
+:::
+
+The sub-directory `staff/` will be populated with a folder for each of the following: instructors, co-instructors, and TAs. The `students/` sub-directory contains a folder for each student. And the `data/` sub-directory can be populated by those in staff but is read-only for students.
+
+All those in staff have read-write-execute permissions within the entirety of their courses directory, allowing them to store data, homework assignments, build conda environments, create new directories, etc, as they see fit.
+
+Each course directory gets a default 1TB of storage space. This amount can be increased in the initial application form for classroom access, or requested anytime during an actively running course, by contacting <rchelp@northeastern.edu>
+
+Once the course has ended, and final grades have been submitted, the courses space including all data and shared class files will be archived, and all student personal directories will be deleted. Any students who had access to the HPC cluster only though the course will no longer have access when the course is completed.
+
+Please see our page on {ref}`getting-access` if you would like an account that persists through taking courses.
+
+## Courses Partitions
+---
+We have two partitions dedicated to the use of students and instructors for the duration of their course.
+
+
+
+::::{list-table}
+---
+header-rows: 1
+---
+* - Name
+  - Time Limit (default/max)
+  - Running Jobs (max)
+  - RAM Limit
+* - courses
+  - 4 hr / 24 hrs
+  - 50
+  - 256 GB
+* - courses-gpu
+  - 4 hr / 8 hrs
+  - 8
+  - 12 GB
+::::
+
+
+The resources available in the courses/courses-gpu partitions can be queried with the command `sinfo` as run in the command line. We manage the resourses in courses/courses-gpu each term in response to the number of courses and requested usage per course.
+
+:::{code-block} bash
+sinfo -p courses-gpu  --Format=nodes,cpus,gres,statecompact
+:::
+
+
+These partitions can be used in the following ways:
+
+(sbatch-courses-index)=
+### sbatch script
+
+An sbatch script can be submitted on the command line via the command `sbatch scriptname.sh`. Below are some examples of sbatch scripts using the courses and courses-gpu partitions. See {ref}`slurm-running-jobs` for more information on running sbatch scripts or run `man sbatch` for additional sbatch parameters.
+
+::::{dropdown} courses partition
+:::{code-block}bash
+
+#!/bin/bash
+
+#SBATCH --nodes=1  
+#SBATCH --time=4:00:00  
+#SBATCH --job-name=MyJobName  
+#SBATCH --partition=courses  
+#SBATCH --mail-type=ALL  
+#SBATCH --mail-users=$USER@northeastern.edu  
```

Comment:
We should put a note that $USER needs to be adjusted and this doesn't work as expected as with unix (I found this out a couple of weeks ago).